src/components/Box/Box.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import Button from '@mui/material/Button';
+
+export default function BoxSx({ content, space, handleBoxClick, active }) {
+  const handleClick = () => {
+    handleBoxClick({ content, space });
+  };
```

Comment:
This could have just been called in the onClick. if you have time to refactor try and update it so that it all lives in the onClick.